src/AST/Translator.Common.dfy

```diff
@@ -0,0 +1,37 @@
+include "../Interop/CSharpDafnyASTModel.dfy"
+include "../Interop/CSharpDafnyInterop.dfy"
+include "../Utils/Library.dfy"
+
+module Bootstrap.AST.Translator.Common {
+  import opened Utils.Lib.Datatypes
+  import opened Interop.CSharpDafnyInterop
+  import C = Interop.CSharpDafnyASTModel
+
+  datatype TranslationError =
+    | Invalid(msg: string)
+    | GhostExpr(expr: C.Expression)
+    | UnsupportedType(ty: C.Type)
+    | UnsupportedExpr(expr: C.Expression)
+    | UnsupportedStmt(stmt: C.Statement)
```

Comment:
We could get rid of this now that it's no longer used. And we could _almost_ get rid of `UnsupportedExpr`, too.